### ISSUE TITLE
TypeError: Cannot read property 'map' of undefined

### ISSUE THREAD

**Issue by joshaber** [desktop/desktop#1557]
https://haystack.githubapp.com/desktop/needles/msrjQGVfU8gtlzG5KFCFdQ

```
TypeError: Cannot read property 'map' of undefined
at Pe._loadFromSharedProcess (file:///app/renderer.js:3:318648)
at I.Dispatcher.onSharedDidUpdate (file:///app/renderer.js:3:340325)
at EventEmitter.z.ipcRenderer.on (file:///app/renderer.js:3:339614)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
```

**Comment by niik** [comment]
I believe this was fixed in #1523, prior to that the emails field in `Account` could end up being an empty string. Also xref #1522

**Comment by joshaber** [comment]
🆒 🍪
